modules/virt-updating-multiple-vms.adoc

```diff
@@ -0,0 +1,87 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-edit-vms.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-updating-multiple-vms_{context}"]
+= Updating multiple virtual machines
+
+You can update multiple virtual machines (VMs) at the same time by using the command line.
+
+.Prerequisites
+
+* You installed the `oc` CLI.
+* You have access to the {product-title} cluster with `cluster-admin` permissions.
+
+.Procedure
+
+. Create a privileged service account by running the following commands:
```

Comment:
More context for what each command might be useful to give a customer onus on the importance of the sequence of commands.